rotkehlchen/api/rest.py

```diff
@@ -5409,3 +5409,62 @@ def force_refetch_evm_transactions(
             transaction_count += (after_count - before_count)
 
         return _wrap_in_ok_result({'new_transactions_count': transaction_count})
+
+    def addresses_interacted_before(
+            self,
+            from_address: ChecksumEvmAddress,
+            to_address: ChecksumEvmAddress,
+    ) -> Response:
+        with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
+            cursor.execute(
+                'SELECT COUNT(*) FROM history_events JOIN evm_events_info ON '
+                'history_events.identifier=evm_events_info.identifier WHERE '
+                'location_label=? AND address=?',
+                (from_address, to_address),
```

Comment:
this check is not bidirectional as it could be the `address` interacting with `location_label`. 
on a second thought, this might not be what you're trying to achieve.

Reply:
yes, intentionally is not bidirectional since you want only the check in the direction that you send. Otherwise someone could send you a transaction and then trick you to send back to that address.